setup.py

```diff
@@ -16,8 +16,8 @@
     description = 'prophy: fast serialization protocol',
     long_description = long_description,
     packages = find_packages(),
-    requires = ['ply'],
-    install_requires = ['ply'],
+    requires = ['ply', 'renew'],
+    install_requires = ['ply', 'renew==0.2'],
```

Comment:
Is it deliberate to depend only on `0.2`? This restricts user from combining prophy `1.2.0` and renew `0.3`.

Reply:
It's to avoid any interface change in `renew`, which is still a quite young project. It would like to keep it fixed. E.g. name of one of (only two) exposed functions in `renew` just changed from `reproduction_string` to `reproduction` in `v.0.3`.

Reply:
I've just updated the requirement to `renew>=0.3.0`. Let me know if it's acceptable for you. We cannot use anything lower than 0.3.0 because its interface changed at that point.

Reply:
Sure - it's ok.